repos/spack_repo/builtin/packages/rivet/package.py

```diff
@@ -154,6 +157,14 @@ def configure_args(self):
             args += ["--with-hepmc3=" + self.spec["hepmc3"].prefix]
             args += ["--with-hepmc3-libpath=" + self.spec["hepmc3"].libs.directories[0]]
 
+        val = self.spec.variants["plugin-match"].value
+        if val != "none":
+            args += [f"--with-plugin-match={val}"]
```

Comment:
If the variant is defined with `multi=True` then it can take comma-separated arguments and we can turn them into the space-separated patterns that rivet expects with
```suggestion
            args += [f"--with-plugin-match={' '.join(val)}"]
```
Tested and works for `plugin-match=HERA,LEP,MC`.